Tests/ResponseTests.swift

```diff
@@ -311,9 +311,9 @@ class ResponseJSONTestCase: BaseTestCase {
             XCTAssertNotNil(response?.metrics)
         }
 
-        // The `as NSString` cast is necessary due to a compiler bug. See the following rdar for more info.
-        // - https://openradar.appspot.com/radar?id=5517037090635776
-        if let form = (response?.result.value as AnyObject?)?["form" as NSString] as? [String: String] {
+        if
+            let responseDictionary = response?.result.value as? [String : Any],
+            let form = responseDictionary["form"] as? [String : String] {
```

Comment:
Addressed in [a0337](https://github.com/Alamofire/Alamofire/pull/2163/commits/03dbf047a5fd956e850a135efbf651c390ea0337).